docs/dev-guide/web.md

```diff
@@ -7,21 +7,88 @@ sidebar_label: Start
 Welcome to the developers guide for web! This guide will help you setup a development
 environment to start working on the Jitsi Meet codebase.
 
-## Building the sources
+
+## Prerequisites
+
+The setup for the development environment uses `make`, a native command from GNU/Linux. **This guide assumes that you have a Linux system with Ubuntu or Debian, or Windows Subsystem for Linux (WSL)[https://en.wikipedia.org/wiki/Windows_Subsystem_for_Linux]**. If you have iOS or other Linux distro, the process should of the installation of packages will be diferent depending on your OS.
 
 :::note
 Node.js >= 12 and npm >= 6 are required.
 :::
 
-On Debian/Ubuntu systems, the required packages can be installed with:
-- Download "Linux Binaries (x64)" from https://nodejs.org/en/download/
-- Install Node.js following these instructions: https://github.com/nodejs/help/wiki/Installation
+1. Install `nodejs` and `npm`
+
+The easiest way to install `nodejs` and `npm` is to run
 
-To build the Jitsi Meet application, just type
+```bash
+sudo apt-get install nodejs npm
 ```
-make
+
+Verify that the version of `nodejs` is >= 12 and the version of `npm`>= 6 with
+
+```bash
+node -v && npm -v
+```
+
+A result that looks good is, for example,
+
+```
+v12.18.3
+6.14.6
+```
+
+2. Install `npm` dependencies
+
+Run
+
+```bash
+npm update
+npm install webpack
```

Comment:
This is also not needed, webpack is a dev dependency and gets installed when running npm install.

Reply:
Maybe a dedicated section titled "Developing on Windows" would be best.